src/histolab/filters/image_filters.py

```diff
@@ -581,7 +809,12 @@ class LocalOtsuThreshold(ImageFilter):
     -------
     np.ndarray
         NumPy boolean array representing the mask based on local Otsu threshold
-    """
+
```

Comment:
missing example

Reply:
I want to fix this (issue #255) before adding the example

Reply:
alright great